### PR TITLE
Add EliiseS/devcontainer features to the list

### PR DESCRIPTION
**Pull request by EliiseS**
Let me know if I've done anything incorrectly :) 

I'll be adding at least one more additional feature shortly.

### CODE REVIEW

File: _data/collection-index.yml [hunk 1]
```diff
@@ -218,3 +218,8 @@
   contact: https://github.com/mamba-org/devcontainer-features/issues
   repository: https://github.com/mamba-org/devcontainer-features
   ociReference: ghcr.io/mamba-org/devcontainer-features
+- name: Devcontainer features by @EliiseS
+  maintainer: EliiseS
+  contact: https://github.com/EliiseS/devcontainer-features/issues
+  repository: https://github.com/EliiseS/devcontainer-features
+  ociReference: ghcr.io/EliiseS/devcontainer-features
```
Comment (samruddhikhandale) [inline _data/collection-index.yml]:
```suggestion
  ociReference: ghcr.io/eliises/devcontainer-features
```

The ghcr packages supports lower casing, see here --> https://github.com/EliiseS/devcontainer-features/pkgs/container/devcontainer-features%2Fbash-profile


image

**Review by samruddhikhandale — CHANGES_REQUESTED** [review]
Thanks, left one comment.

File: _data/collection-index.yml (hunk 1 above)
Comment (EliiseS) [reply _data/collection-index.yml]: Thanks! Committed your suggestion 😄

**Review by samruddhikhandale — APPROVED** [review]
Awesome, thanks!

**Pull request merged**